### PR TITLE
[Bugfix] Format and code correction

### PR DESCRIPTION
**Pull request by NoftScarlet**
### Note:
As noted in the closed PR [here](https://github.com/PDFTron/webviewer-react/pull/1), I am re-creating separate branches for fixes to `src` and updates to `examples`. 

### Fixed issues:
This branch addresses the requested changes in this [PR](https://github.com/PDFTron/webviewer-react/pull/1)
This branch was checked out from branch 'bugfix/formats' up to commit fd0d6a1 (3 commits after my last reply to @lbittner-pdftron 's comment in the [PR](https://github.com/PDFTron/webviewer-react/pull/1), plus cherry pick of 423364c and 69a1a0a. 

Updates to examples code will be made in a separate branch after this PR passes review.

### Other:
Tests / Linting setup are taken care of in separate branches. After all the repo constructional branches (check current branch list for details) are merged, this repository will have a more standardized Issue / PR / Testing process.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,15 @@
   "main": "src/index.ts",
   "types": "src/index.d.ts",
   "scripts": {
-    "startdv": "npm run build && esbuild --bundle examples/DocumentViewer/index.js --loader:.js=jsx --outfile=www/main.js --servedir=www",
-    "startctx": "npm run build && esbuild --bundle examples/Context/index.js --loader:.js=jsx --outfile=www/main.js --servedir=www",
-    "ts-types": " tsc -d --emitDeclarationOnly --outDir lib/context",
     "build": "node ./esbuild.js && npm run ts-types",
-    "preparevwlib": "node copy-webviewer-file.js"
+    "deploydemo": "npm run beforestart && esbuild --bundle examples/Context/index.js --loader:.js=jsx --outfile=www/main.js && firebase deploy --only hosting:pdftron-ychen02",
```
Comment (lbittner-pdftron) [inline package.json]: I don't think this script should exist in a public repo

File: package.json [hunk 2]
```diff
@@ -22,16 +26,18 @@
...
     "esbuild": "^0.14.1",
     "fs-extra": "^10.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
     "typescript": "^4.5.2"
   },
+  "peerDependencies": {
+    "@pdftron/webviewer": "^8.2.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
```
Comment (lbittner-pdftron) [inline package.json]: These should be dev dependencies

File: src/components/DocumentVIewerSImpleDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// @ts-ignore
```
Comment (lbittner-pdftron) [inline src/components/DocumentVIewerSImpleDisplay.tsx]: Please check your spelling in the file name, there are two misplaces capital `I`'s

Comment (lbittner-pdftron) [reply src/components/DocumentVIewerSImpleDisplay.tsx]: Also does the ts-ignore need to be there?

File: src/context/useInstance.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { DocumentViewerContext } from './Context'
+import { useContext } from 'react';
+import { TContextState } from './Context';
+
+export default function useInstance(): TContextState{
+  const { instance, setInstance } = useContext(DocumentViewerContext);
+  return { instance, setInstance }
```
Comment (lbittner-pdftron) [inline src/context/useInstance.ts]: This should be memoized

File: src/utils/index.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import { DISABLED_ELEMENTS } from "./utils";
+
+export {DISABLED_ELEMENTS}
```
Comment (lbittner-pdftron) [inline src/utils/index.ts]: I don't really think this file is necessary

**Review by lbittner-pdftron — CHANGES_REQUESTED** [review]

File: src/context/useInstance.ts (hunk 4 above)
Comment (lbittner-pdftron) [reply src/context/useInstance.ts]: Or you can just do `return useContext(DocumentViewerContext);`, then you don't need to memoize

File: package.json (hunk 1 above)
Comment (NoftScarlet) [reply package.json]: removed

File: package.json (hunk 2 above)
Comment (NoftScarlet) [reply package.json]: fxied

File: src/components/DocumentVIewerSImpleDisplay.tsx (hunk 3 above)
Comment (NoftScarlet) [reply src/components/DocumentVIewerSImpleDisplay.tsx]: fixed

File: src/context/useInstance.ts (hunk 4 above)
Comment (NoftScarlet) [reply src/context/useInstance.ts]: fixed

File: src/utils/index.ts (hunk 5 above)
Comment (NoftScarlet) [reply src/utils/index.ts]: removed

**Comment by NoftScarlet** [comment]
Apart from the issues above I also had additions and changes to DocumentViewer.tsx based on what linter complained.

**Review by lbittner-pdftron — APPROVED** [review]

**Pull request merged**